osc-staging.py

```diff
@@ -331,14 +331,19 @@ def do_staging(self, subcmd, opts, *args):
 
             if version_openqa == version_totest and not totest_dirty:
                 cmd = AcceptCommand(api)
+                accepted_prjs = []
                 for prj in args[1:]:
-                    if not cmd.perform(api.prj_from_letter(prj), opts.force):
+                    if cmd.perform(api.prj_from_letter(prj), opts.force):
+                        accepted_prjs.append(prj)
```

Comment:
fine with me, but that might generate multiple times commits per one time accepting, ie. accepting 5 staging project will committing 5 times. is it ok for you?